packages/run-protocol/scripts/init-core.js

```diff
@@ -0,0 +1,75 @@
+/* global process */
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+export const defaultProposalBuilder = async ({ publishRef, install }) => {
```

Comment:
Note: we can continue to tweak this until we actually deploy the proposal (at the end of the week).